src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -56,14 +57,15 @@ export default class SourceFileLinter {
 
 	constructor(
 		context: LinterContext, resourcePath: ResourcePath,
```

Comment:
Maybe [parameter properties](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties) would come in handy for constructors like this one

Reply:
Thanks!
Very convenient, indeed!